doc/sphinx/_static/js/get_options.js

```diff
@@ -1,4 +1,72 @@
 WINDOW_CONTENTS = window.location.href.split('/')
+SPLIT_REGEX = /([a-zA-Z]+)|([0-9]+)/g;
+REPLACE_REGEX = /[^a-zA-Z0-9]*/g
+
+// this function compares the individual version specifiers of two different versions.
+// If were comparing versions 1.0.0rc1 vs 1.1.2. This function compares
+// 1 compared to 1
+// 0 compared to 1
+// 0rc1 vs 2
+// the key here is that it removes all non alpha-numeric numbers, then splits by character TRANSITION
+// Then it compares 1-1 for THOSE.
+function comparePreviewSpecifer(v1, v2){
+    var v1parts = v1.replace(REPLACE_REGEX, '').split(SPLIT_REGEX).filter(a => a)
```

Comment:
One additional change i can think of is "if(v1 === v2) return 0" in the beginning to avoid further checks if both values are same.

Reply:
Yeah good call.